docs/development/documentation_preview.rst

```diff
@@ -4,11 +4,12 @@
 Documentation Preview
 *********************
 
-Most of the time it's enough to build the documentation locally and see how things are going. But sometimes 
-it's nice to share our changes and get some feedback from other collaborators. 
+To preview your changes to the documentation please:
 
-To preview your changes to the documentation first make sure you have GitHub Actions activated on your fork
-(see the Actions tab). Then, there are two ways to trigger the build:
+#. Enable GitHub Actions to run on your fork (see the *Actions* tab).
+#. Make sure your fork is deploying GitHub Pages inside the root of ``gh-pages`` branch (see the *Settings* tab).
+
+Then, there are two ways to trigger the build:
 
 #. If the branch you are working on contains the word ``doc`` in it, then every commit will trigger the build.
 #. Add the tag ``[build docs]`` on the commit message before pushing.
```

Comment:
I'd also suggest to rephrase it as "If your commit message contains (or is prefixed with?) ``[build docs`` tag, then that commit will trigger the build."

Reply:
And at the end we can write a note that if you forgot to follow any of the above two ways, you can trigger the build by pushing an empty commit with `[build docs]` message to your branch, as follows: `git commit --allow-empty -m "[build-docs]"`.

I've been telling this in 2-3 recent PR where neither author's branch contained "doc", nor they added `[build docs]` to their last commit - so it's good to have it mentioned in docs.

Reply:
I agree to all these comments! Sometimes I use `build_docs` or `build-docs` and that does not work :joy: maybe we can think a way to have multiple commit tags for this in the future @aribalam .

Reply:
Done.

Reply:
Also added new valid tags `[build docs] [build_docs] [build doc] [build_doc]` @jaladh-singhal